docs/source/command_line.rst

```diff
@@ -304,6 +305,11 @@ Here are some more useful flags:
   ``explicit`` disallows explicit ``Any`` in type positions such as type
   annotations and generic type parameters.
 
+  ``generics`` disallows usage of generic types that do not specify explicit
+  type parameters. Moreover, built-in collections (such as ``list`` and
+  ``dict``) become disallowed as you should use their aliases from the typing
+  module (such as ``List`` and ``Dict``).
```

Comment:
I would say here ``List[int]`` and ``Dict[str, str]``, since IIUC plain ``List`` and ``Dict`` are prohibited with this option as well.